manuscript/the-meat.txt

```diff
@@ -188,20 +188,20 @@ You should now see a histogram like this:
 
 ## Adding an axis
 
-Axes are one of my favourite feats of d3.js magic. You just call a function, set some parameters and bam, you’ve got an axis.
+Axes are one of my favourite feats of d3.js magic. You simply call a function, set some parameters, and **BAM**, you’ve got an axis.
```

Comment:
Fixed that first part.

That second part: noted. I was treading lightly when it came to the book. :)